test/passes/remove-unused-module-elements.txt

```diff
@@ -248,7 +247,4 @@
    )
   )
  )
- (func $starter (; 5 ;) (type $2)
```

Comment:
Good idea to optimize this!

Code looks good, but for testing, I think this case here that is now optimized out might be the only case we have for the start function. Please add another module in this test with a non-empty start function, to make sure we test that case too.

Reply:
Sure, going to add one!